src/test/java/domain/AttendanceManagerTest.java

```diff
@@ -0,0 +1,219 @@
+package domain;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.assertj.core.api.Assertions.assertThatThrownBy;
+import static org.junit.jupiter.api.Assertions.assertAll;
+
+import java.util.List;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Nested;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.CsvSource;
+import util.Current;
+import util.DateUtil;
+
+public class AttendanceManagerTest {
+    @ParameterizedTest
+    @DisplayName("닉네임이 등록되었는지 확인한다")
+    @CsvSource(value = {"후우, true", "마후우, false"})
+    void should_return_true_when_registered_nickname(NickName nickName, boolean expected) {
+        // given
+        NickName registedNickName = new NickName("후우");
+        AttendanceManager attendanceManager = new AttendanceManager();
+        attendanceManager.register(registedNickName);
+
+        // when
+        boolean result = attendanceManager.isRegistered(nickName);
+
+        // then
+        assertThat(result).isEqualTo(expected);
+    }
+
+    @Test
+    @DisplayName("등록되지 않은 닉네임을 입력하면 예외가 발생한다")
+    void should_throw_exception_when_not_registered_nickname() {
+        // given
+        NickName nickName = new NickName("후우");
+        AttendanceRecord attendanceRecord = AttendanceRecord.of("11", "10:00");
+        AttendanceManager attendanceManager = new AttendanceManager();
+
+        // when & then
+        assertAll(() -> assertThatThrownBy(() -> {
+            attendanceManager.attend(nickName, attendanceRecord);
+        }).isInstanceOf(IllegalArgumentException.class), () -> assertThatThrownBy(() -> {
```

Comment:
```suggestion
        assertAll(
                () -> assertThatThrownBy(() -> attendanceManager.attend(nickName, attendanceRecord))
                        .isInstanceOf(IllegalArgumentException.class),
```

여기도 이런식으로 줄바꿈을 변경하면 훨씬 읽기 좋겠네요.

Reply:
IDE의 코드 정렬 기능을 연속해서 2번 사용하면 한 줄에 120자 이내로 이상하게 코드가 정렬이 되네요.
커밋 전 코드 줄바꿈 상태를 확인하고 커밋 하겠습니다.
컨벤션 오류 찾아주셔서 감사합니다. 👍